### PR TITLE
implementing checkpoint3

### PR DESCRIPTION
**Pull request by JaymesKat**
#### What does this PR do?
Implements tasks in checkpoint 3 for Kampala Cohort 7

#### Description of Task to be completed?
- Secure all endpoints with JWT token authorization
- Use postgresql database for data persistence

#### How should this be manually tested?
- clone repository
- install postgres on your local machine
- Create a database for project and set environment variables for database: `POSTGRES_USER`, `POSTGRES_PW`, `POSTGRES_URL`, `POSTGRES_DB`, `POSTGRES_TEST_DB`, 
- Run database setup and migration commands: 
   - `python manage.py db init`
   - `python manage.py db migrate`
   - `python manage.py db upgrade`
   - `python manage.py resetdb`
- Run application `python run.py` and test endpoints with Postman, curl or preferred client

#### Any background context you want to provide?
This builds incrementally on the checkpoint 2 of this project that used non-persistent data storage

#### What are the relevant pivotal tracker stories?
[#158288531](https://www.pivotaltracker.com/story/show/158288531)

#### Screenshots (if appropriate)
API endpoint descriptions
screen shot 2018-07-04 at 12 29 48
screen shot 2018-07-04 at 12 30 09

### CODE REVIEW

File: app/resources/v1/auth.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
+
+import pdb
```
Comment (Descartess) [inline app/resources/v1/auth.py:2]: Why is this here

Comment (Descartess) [reply app/resources/v1/auth.py:2]: Try and resolve merge conflicts too

Comment (JaymesKat) [reply app/resources/v1/auth.py:2]: I was debugging and left the pdb lying there. Thanks for pointing that out

**Comment by JaymesKat** [comment]
@Descartess I have managed to resolve merge conflicts
